File: 2020W53/bbuyo/programmers_두개뽑아서더하기.java
import java.util.ArrayList;

class Solution {
    public int[] solution(int[] numbers) {
        ArrayList<Integer> rslt = new ArrayList<>();
        int len = numbers.length;
        for(int i=0; i<len; i++) {
            for(int j=i+1; j<len; j++) {
                int tmpSum = numbers[i]+numbers[j];
                rslt.add(tmpSum);
            }
        }

        int[] result = rslt.stream().distinct().sorted().mapToInt(i->i).toArray();
        return result;
    }
}

File: 2020W53/bbuyo/programmers_두개뽑아서더하기.py
import itertools

def solution(numbers):
    answer = []
    a = set(itertools.permutations(numbers, 2))

    for i in a:
        if i[0] in numbers and i[1] in numbers:
            answer.append(sum(i))

    answer = list(set(sorted(answer)))
    return answer